### ISSUE TITLE
Support pair type

### ISSUE THREAD

**Issue by alexmingoia** [nikita-volkov/hasql-postgres#26]
The `pair` type is commonly used to store geocoordinates in Postgres using `GEOGRAPHIC(pair)`. It'd be useful to add support for this.

**Comment by nikita-volkov** [comment]
I suppose the support for composite types, which is coming in the new version, shall cover this, but we'll see. This is an extension type, right?

**Comment by alexmingoia** [comment]
No, its part of postgres not an extension http://www.postgresql.org/docs/9.4/static/datatype-geometric.html#AEN6730 - theres actually quite a few other geometric types as well so maybe this issue should really be for the entire set of geometric types...

**Comment by nikita-volkov** [comment]
Okay. Consider it qualified as such.
